Makefile

```diff
@@ -4,7 +4,10 @@
 #
 ####################################################################
 
+verifyTag:
+	./Tools/verifyTag.sh
+
 tagCommit:
 	git tag `cat sdk-version.txt`; git push origin `cat sdk-version.txt`
 
-deploy: tagCommit
+deploy: verifyTag tagCommit
```

Comment:
This is good safety net. Little confused with name. Can we rename "deploy" to "verifyTagAndCommit" ?

Reply:
The name is coming from the fact that this is what we are doing for deployment. If you look at the iOS makefile, it also updates the constants file. We can make another task called `verifyTagAndCommit` but I want the deployment to be a single make deploy